src/Apps/Consign/Routes/MarketingLanding/Components/LandingPage/HeaderSWA.tsx

```diff
@@ -38,8 +41,10 @@ export const HeaderSWA = () => {
 
   const trackGetInTouchClick = () => {
     trackEvent({
-      action: "clickedGetInTouch",
-      context_module: "Header",
+      action: ActionType.tappedConsignmentInquiry,
```

Comment:
@audrey-thenot "clickedGetInTouch" action was used on the old screens, but it's not in cohesion. Let me know, please, if it'd be best to keep using it instead of the one from cohesion

Reply:
It seems that we are trying to move everything to Cohesion, so I would say let's take the one from Cohesion. I'll update the analytics pipeline accordingly.

Reply:
Alright
And we're still keeping Header and Footer not important from cohesion, right? In cohesion they are header and footer (not capital letter)

Reply:
yes